docs/guide/tooltip.md

```
@@ -0,0 +1,81 @@
+            }
+        }
+    )
+    + save("docs/iframes/tooltip-custom-css2.html", minify=True)
+)
+```
+
+<iframe width="100%" height="600" src="../iframes/tooltip-custom-css2.html" style="border:none;"></iframe>
+
+!!! tip
+
+      AI is crazy good at CSS! Just describe what you want and it'll be able to do it for sure.
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)